scripts/GetLocalizedFiles.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+id=""
+alias=""
+password=""
+outputDirectory=""
+renameLanguageFolder=true
+extension=""
+parserId=0
+isoauth=false
+tokenServer="tdb-touchdownbuild-prod"
+
+function ParseArgs()
+{
+while getopts "nut:a:p:f:r:o:e:s:w:" arg
+do
+case "$arg" in
+n)
+renameLanguageFolder=false;;
+u)
+isoauth=true;;
+t)
+id=$OPTARG;;
+a)
+alias="$OPTARG";;
+p)
+password="$OPTARG";;
+f)
+filePath="$OPTARG";;
+r)
+relativeFilePath="$OPTARG";;
+o)
+outputDirectory="$OPTARG";;
+e)
+extension="$OPTARG";;
+s)
+parserId="$OPTARG";;
+w)
+tokenServer="$OPTARG";;
+-)      break;;
+esac
+done
+}
+
+ParseArgs $*
+
+echo Team ID: $id
+echo Alias: $alias
+echo File Path: $filePath
+echo Relative Path: $relativeFilePath
+echo Parser Id: $parserId
+echo Output Directory: $outputDirectory
+if($isoauth = true); then
+echo "using oauth."
+else
+echo "using NTLM."
+fi
+
+# parse json and return value of the key
+function jsonValue() {
+KEY=$1
+num=$2
+jsonParseCmd=`awk -F"[,:}]" '{for(i=1;i<=NF;i++){if($i~/'$KEY'\042/){print $(i+1)}}}' | tr -d '"' | sed -n ${num}p`
```

Comment:
yay, more awk

Reply:
Yeah this comes from the original script from the localization team, not interested in rewriting/clarifying it.